AmplifyTestCommon/Helpers/AuthHelper.swift

```diff
@@ -13,8 +13,8 @@ class AuthHelper {
         let callbackInvoked = DispatchSemaphore(value: 0)
 
         AWSMobileClient.default().initialize { userState, error in
-            if let error = error {
-                fatalError("Error initializing AWSMobileClient. Error: \(error.localizedDescription)")
+            if let error = error as? AWSMobileClientError {
```

Comment:
Why only fatal on AWSMobileClientErrors?

Reply:
i didn't think AWSMobileClient returns anything other than AWSMobileClientErrors, so i was casting to it and extracting the `message` out for a friendly local debugging message

Reply:
I follow your logic, but it relies a lot of knowledge of the code paths that lead up to this, which in AWSMobileClient is a little fraught right now. Consider a pattern that logs the `message` if it's an AWSMobielClientError and the `localizedDescription` otherwise.

Reply:
adding check for AWSMobileClientError, else fatalError with localizedDescription